src/ctia/documentation/routes.clj

```diff
@@ -75,7 +77,7 @@
 (defn render-request
   "read the requested file from resources, render it if needed"
   [path-info]
-  (let [file-path (subs path-info 1)
+  (let [file-path (str doc-resource-prefix path-info)
         file-content (get-file-content file-path)
```

Comment:
Could you tighten the security of the function `get-file-content` in the same way [`ring.util.response/resource-response`](https://github.com/ring-clojure/ring/blob/master/ring-core/src/ring/util/response.clj#L324) is? It avoids:
- directory traversal
- it doesn't follow symlinks if specified